### ISSUE TITLE
Gazebo simulation

### ISSUE THREAD

**Issue by dylanvaughn** [robomakery/pvc-powerchair#20]
basic gazebo simulation

**Comment by dylanvaughn** [comment]
@bus710 to provide more details

**Comment by bus710** [comment]
I am thinking to use Onshape to draw a new design for simulation so that I was looking for export option for DAE file. Althohg Onshape doesn't support DAE, STL is in the list. Also fortunately, There are bunch of apps and services, which provide converting between STL and DAE. 

I am gonna draw some design by using Onshape and make some shapes in Gazebo this weekend.

**Comment by dylanvaughn** [comment]
@bus710 it probably makes sense to just go ahead and make a URDF model for the chair to use in Gazebo - we are already tracking that task in #13.  I put that into this milestone.

**Comment by bus710** [comment]
@dylanvaughn, that's true... I am editing one of my previous model to adjust for Gazebo 7. (Because the sdf was design for Gazebo 1.7 and makes some crash for Gazebo 7.)

**Comment by bus710** [comment]
Although I just put three objects (body and two big wheel) yet, I think that I can build the first model in this weekend.  

![default_gzclient_camera 1 -2016-06-17t17_35_05 360973](https://cloud.githubusercontent.com/assets/8920680/16168004/3712afba-34b2-11e6-9174-e30582c8c3c1.jpg)

**Comment by LeviLister** [comment]
Looks like a good start!

**Comment by bus710** [comment]
I issued a PR for simulation to this repo.
Please let me know if I can merge my branch.

**Comment by dylanvaughn** [comment]
@bus710 I left some comments on the PR

**Comment by bus710** [comment]
@dylanvaughn, Yes, I saw your mention and it was very sincere. I will fix the PR soon.. Thank you!
- You only need to run the script '01_cp_models' once after you clone the repo or edit models. Otherwise we need to set an environment variable in the ROS system.
